travis/before_script.osx-clang.sh

```diff
@@ -2,4 +2,4 @@
 set -ev
 mkdir sleef.build
 cd sleef.build
-cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=../install -DSLEEF_SHOW_CONFIG=1 -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl -DENFORCE_TESTER3=TRUE -DBUILD_QUAD=TRUE ..
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=../install -DSLEEF_SHOW_CONFIG=1 -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl -DENFORCE_TESTER3=TRUE -DBUILD_QUAD=TRUE -DBUILD_INLINE_HEADERS=TRUE ..
```

Comment:
Do I understand correctly that you are always generating the header files because you want to test them on each architecture with the `iuti*` executables? Has this increased the running time in CI?

Reply:
Yes, running time in CI is increased.
If that is a problem, we can just turn off generation.